features/step_definitions/networking.rb

```diff
@@ -336,46 +203,27 @@
   _host = _node.host
   _admin = admin
 
-  if env.version_lt("3.7", user: user)
-    @result = _admin.cli_exec(:get, resource: "clusternetwork", resource_name: "default", template: "{{.network}}")
-  else
-    @result = _admin.cli_exec(:get, resource: "clusternetwork", resource_name: "default", template: '{{index .clusterNetworks 0 "CIDR"}}')
-  end
-  unless @result[:success]
-    raise "Can not get clusternetwork resource!"
-  end
-
-  subnet = @result[:response]
-
-  if env.version_lt("3.6", user: user)
-    @result = _host.exec('iptables -D INPUT -p udp -m multiport --dports 4789 -m comment --comment "001 vxlan incoming" -j ACCEPT')
-    raise "failed to delete iptables rule #1" unless @result[:success]
-    @result = _host.exec('iptables -D INPUT -i tun0 -m comment --comment "traffic from SDN" -j ACCEPT')
-    raise "failed to delete iptables rule #2" unless @result[:success]
-    @result = _host.exec("iptables -D FORWARD -d #{subnet} -j ACCEPT")
-    raise "failed to delete iptables rule #3" unless @result[:success]
-    @result = _host.exec("iptables -D FORWARD -s #{subnet} -j ACCEPT")
-    raise "failed to delete iptables rule #4" unless @result[:success]
-    @result = _host.exec("iptables -t nat -D POSTROUTING -s #{subnet} -j MASQUERADE")
-    raise "failed to delete iptables nat rule" unless @result[:success]
-  else
-    @resule = _host.exec('iptables -D OPENSHIFT-FIREWALL-ALLOW -p udp -m udp --dport 4789 -m comment --comment "VXLAN incoming" -j ACCEPT')
-    raise "failed to delete iptables rule #1" unless @result[:success]
-    @resule = _host.exec('iptables -D OPENSHIFT-FIREWALL-ALLOW -i tun0 -m comment --comment "from SDN to localhost" -j ACCEPT')
-    raise "failed to delete iptables rule #2" unless @result[:success]
-    @resule = _host.exec("iptables -D OPENSHIFT-FIREWALL-FORWARD -d #{subnet} -m comment --comment \"forward traffic from SDN\" -j ACCEPT")
-    raise "failed to delete iptables rule #3" unless @result[:success]
-    @resule = _host.exec("iptables -D OPENSHIFT-FIREWALL-FORWARD -s #{subnet} -m comment --comment \"forward traffic to SDN\" -j ACCEPT")
-    raise "failed to delete iptables rule #4" unless @result[:success]
-
-    # compatible with different network plugin
-    @result = _host.exec("iptables -S -t nat \| grep '#{subnet}' \| cut -d ' ' -f 2-")
-    raise "failed to grep rule from the iptables nat table!" unless @result[:success]
-    nat_rule = @result[:response]
-
-    @resule = _host.exec("iptables -t nat -D #{nat_rule}")
-    raise "failed to delete iptables nat rule" unless @result[:success]
-  end
+  step "The subnet from the clusternetwork resource is stored in the clipboard"
+  subnet = cb.clusternetwork
+
+  @resule = _host.exec('iptables -D OPENSHIFT-FIREWALL-ALLOW -p udp -m udp --dport 4789 -m comment --comment "VXLAN incoming" -j ACCEPT')
```

Comment:
@rbratta typo in instance variable name ```resule```  everywhere? You are using ```resule``` but checking status of ```result``` at most of the places

Reply:
Also STDERR issue has 3 bugs open under openshift which i don't recall at the moment so we can design logic independent of whether STDERR is included in output or not.

Reply:
> @rbratta typo in instance variable name `resule` everywhere? You are using `resule` but checking status of `result` at most of the places

Looks like the typo was in the old code and I just copy-pasted it.  I think I saw that and assumed it was on purpose.

Thanks for catching this.

Reply:
> Also STDERR issue has 3 bugs open under openshift which i don't recall at the moment so we can design logic independent of whether STDERR is included in output or not.

I'm fine with using `@result[:stdout].to_s`. I'm not sure if the `.to_s` is necessary, haven't figured out what the underlying type of @result[:stdout]` is.